draft-ietf-v6ops-464xlaton.xml

```diff
@@ -227,9 +227,12 @@ While disabling CLAT is impactful for all applications and traffic flows already
 <xref target="Security"/> discusses this aspect in more details.
 </t>
 <t>
-When a node disables CLAT, it might indicate a network misconfiguration or an attack in progress. It may be useful for an administrator to receive signals when CLAT use turns on or off as well as changes to network-received configuration.
-Therefore the node SHOULD log the reason for disabling CLAT and any other changes to CLAT configuration or network signals CLAT is acting on to support administrator debugging and auditing.
-As logging is mostly beneficial in managed environments, the logging behaviour SHOULD be configurable. It MAY be disabled by default to avoid performance impacts when the likelihood of anyone consuming the logs is low.
+Native IPv4 connectivity, which subsequently causes CLAT to be disabled, might not be intended by the administrator.
+It could instead be the result of a network misconfiguration (such as accidentally enabling IPv4) or an attack (such as a rogue DHCPv4 server).
+In such cases it may be useful for an administrator to receive signals when CLAT turns on or off as well as changes to network-received configuration.
```

Comment:
```suggestion
In such cases it might be useful for an administrator to receive signals when CLAT turns on or off as well as changes to network-received configuration.
```

Nit: avoiding normative word collision